docker/docker_image.go

```diff
@@ -23,7 +24,7 @@ func NewDockerImage(img, certPath string, tlsVerify bool) (types.Image, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &Image{Image: image.FromSource(s), src: s}, nil
+	return &Image{Image: image.FromSource(s, []string{manifest.DockerV2Schema1SignedMIMEType, manifest.DockerV2Schema1MIMEType}), src: s}, nil
```

Comment:
This seems a weird place to put this (see also the `FromSource` comments in `image.go`).